manim/config.py

```diff
@@ -89,7 +89,7 @@ def _parse_config(config_parser, args):
 
 args, config_parser, file_writer_config, successfully_read_files = _run_config()
 if _from_command_line():
-    logger.info(f"Read configuration files: {os.path.abspath(successfully_read_files[-1])}")
+    logger.info(f"Read configuration files: {[os.path.abspath(cfgfile) for cfgfile in successfully_read_files]}")
```

Comment:
The reason this was changed was to make the user see only from where the configuration file was read and not the list of all configuration files.

Reply:
`successfully_read_files` will only contain the cfg files that were, well, successfully read. Not all of the config files that are there.

What's the point of only showing the end user the last config file that was read?

They should know what files are being read and what changes each might make to the final config.

Reply:
Agreed with @Aathish04

Reply:
BTW, I know I reviewed and accepted this change in a previous PR, but now that I think of it, I think it should be reverted, as Aathish is doing now.